lib/file_loader.js

```diff
@@ -77,12 +77,16 @@ HttpFileLoader.prototype = _.extend(new FileLoader(), {
   copy: function(suffix, destFile, cb) {
     var srcFile = url.resolve(this.prefix, suffix);
     fsUtil.ensureDirectoryExistsFor(destFile);
-    request(srcFile, opts, function (error, response, body) {
-      if (error || response.statusCode !== 200) {
-        return cb(error || new Error('Unexpected status code: ' + response.statusCode + ' copying ' + srcFile));
+    var r = request(srcFile, opts).on('error', function(error){ return cb(error);}).pipe(fs.createWriteStream(destFile));
+    r.on('response',function(response){
+      if (200 !== response.statusCode){
+        return cb(new Error('Unexpected status code: ' + response.statusCode + ' copying ' + srcFile));
       }
-      fs.writeFile(destFile, body, cb);
     });
+    r.on('error', function(error){
```

Comment:
This is redundant with line 80 and will cause the callback to be called twice on error. It should be removed.